### PR TITLE
Add slate

### PR DESCRIPTION
**Pull request by jorgegilmoreira**

### CODE REVIEW

File: source/includes/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Uphold API Reference
```
Comment (nunofgs) [inline source/includes/README.md]: This file can be removed since it's already at the root.

**Comment by rplopes** [comment]
Rebased this PR so that we can move forward with it. cc @nunofgs

**Comment by rplopes** [comment]
The version of slate we're using is 1.2. The latest available version is 1.4.

I've tried to upgrade from 1.2 to 1.4, but we keep some custom changes that are hard to maintain while upgrading from upstream.

So my suggestion is that we move forward with using 1.2 in this PR, and if later we have more time for upgrades we can consider moving to the latest slate version.

**Comment by rplopes** [comment]
Closing due to inactivity, since this is not a priority for now.

**Pull request closed without merging**